src/components/Calculator.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Calculator.css';
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  render() {
```

Comment:
[OPTIONAL]

- It's okay to disable linters in this stage of the project. But if you want a better way to fix this then I would recommend using `pureComponent` to avoid the linters.